lib/commands/deploynodeapp.js

```diff
@@ -72,6 +80,31 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
```

Comment:
This is for toggling `trireme | edge-services` as defined in the [design doc](https://docs.google.com/document/d/1acE3hyUnDRNuGg3a7eaM-kPkffRSqqdvJZvr4NLMYyQ/edit#). There was offline conversation to change from `backend` to `runtime`.